thoth/package_extract/core.py

```diff
@@ -69,10 +69,11 @@ def extract_image(image_name: str, timeout: int = None, *,
         result = run_analyzers(rootfs_path)
         result['layers'] = layers
 
-    push_gateway = os.getenv('PROMETHEUS_PUSH_GATEWAY', 'pushgateway:9091')
-    if push_gateway:
+    push_gateway_host = os.getenv('PROMETHEUS_PUSHGATEWAY_HOST')
+    push_gateway_port = os.getevn('PROMETHEUS_PUSHGATEWAY_PORT')
+    if push_gateway_host:
         try:
-            pushadd_to_gateway(push_gateway,
+            pushadd_to_gateway(push_gateway_host: push_gateway_port,
```

Comment:
```python
"f{push_gateway_host}:{push_gateway_port}"
```

Reply:
@fridex is that some sort of python magic I dont know about? I see your point and good catch, but wouldnt it be:

```python
pushadd_to_gateway("%s:%s" % (push_gateway_host, push_gateway_port),
```

Reply:
We keep the code base compatible with Python 3.6+ and Python 3.6 offers [f-strings](https://www.python.org/dev/peps/pep-0498/?#rationale). I personally like them.